tests/pkg/flashdb_vfs/kvdb_type_blob_sample.c

```diff
@@ -17,45 +17,42 @@
 
 #define FDB_LOG_TAG "[sample][kvdb][blob]"
 
+extern const char *resstr(bool condition);
+
```

Comment:
Small typo in message of 7dd31e94e36486d5146b2991737fc987c8eb234b: `sampl` -> `sample`